README.md

```diff
@@ -80,8 +77,6 @@ In order to run this project you need:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
 👤 **Nicolas Olaya**
 
 - GitHub: [@NicolasAndrehh](https://github.com/NicolasAndrehh)
```

Comment:
- [ ] Kindly don't forget to implement a [MIT license](https://choosealicense.com/licenses/mit/) because software license tells others what they can and can't do with your source code, so it's essential to make an informed decision. You can also set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). :+1:

Reply:
Okay, I'll do it, thanks for your suggestions!